daprdocs/content/en/operations/performance-and-scalability/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+type: docs
+title: "Performance and Scalability"
+linkTitle: "Performance and Scalability"
+weight: 300
+description: "Benchmarks and guidelines"
+---
+Dapr allows developers to inject sidecars next to their apps that make it easy to write distribured applications. The Dapr APIs allow developers to manage state at scale, trigger their apps from external sources, fetch secrets, publish and consume messages, and call other apps in a secured manner with build in telemetry and resiliency.
+
+Dapr is designed to be highly performant and consume minimal overhead in terms of compute resources.
+
+## System overview
+
+Dapr consists of a data plane (the sidecar that runs next to your app) and a control plane that configures the sidecars and provides additional capabilities like cert management.
+
+### Self-hosted components
```

Comment:
This information should be included https://docs.dapr.io/operations/hosting/self-hosted/self-hosted-overview/ and a link added from here

Reply:
Added link